car/models.py

```diff
@@ -86,6 +87,12 @@ class Color(models.Model):
     class Meta:
         db_table = 'colors'
 
+class ColorType(models.Model):
+    type_name  = models.CharField(max_length=50)
```

Comment:
테이블이름이 color type 이므로 필드명은 name으로만 해도 괜찮을 것 같습니다.
```suggestion
    name  = models.CharField(max_length=50)
```